tests/ccl_test_pyccl_interface.py

```diff
@@ -839,7 +840,7 @@ def test_neutrinos():
 
 def test_redshifts():
     """
-    Test lsst specs module.
+    Test redshifts module.
```

Comment:
Hmm, okay. I'm not sure I am the person to make the call on changing this, especially since it affects all the python tests not just those relevant to this PR. @philbull set up most of the python infrastructure, he may have a comment?